ui/src/jobs/containers/DetailPage.jsx

```diff
@@ -120,7 +123,10 @@ class DetailPage extends Component {
                   <MoreInfo urls={urls} />
                 </Col>
               </Row>
-              <Row type="flex" justify="end">
```

Comment:
As far as I know, the edit button should also be in the job search page

Reply:
Good catch! It was not in the US, I'll update the User Story in Jira and the PR too :+1: